src/ui/ui-types.ts

```diff
@@ -33,6 +35,10 @@ export type PriceBreakdown = {
 };
 
 export type TaxCustomerDetails = {
-  countryCode: string | undefined;
-  postalCode: string | undefined;
+  country: string | undefined;
```

Comment:
Extended these details to support all the needed information to request calculation for the full address.